### PR TITLE
Bee add without nmap

### PR DESCRIPTION
**Pull request by bee-san**
Quiet mode does not print out ports as they are found, leaving users to get impatient.

This adds --without-nmap which prints out ports as they are found but does not run nmap.

A user in John Hammond Discord requested this.

### CODE REVIEW

File: src/scanner/main.rs [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+            ulimit: Some(2_000),
+            command: Vec::new(),
+            accessible: false,
+            scan_order: ScanOrder::Serial,
+<<<<<<< HEAD:src/main.rs
+            no_nmap: false,
+            no_config: false,
+=======
+            ignore_config: false,
+>>>>>>> master:src/scanner/main.rs
```
Comment (bee-san) [inline src/scanner/main.rs:287]: I don't have this in my local branch....

File: src/scanner/main.rs [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+mod scanner;
+use scanner::Scanner;
+
+mod port_strategy;
+use port_strategy::PortStrategy;
+
+use colorful::Color;
+use colorful::Colorful;
+use futures::executor::block_on;
+use rlimit::Resource;
```
Comment (bee-san) [inline src/scanner/main.rs:17]: This is kind of annoying that this entire file is flagged as "new", no idea how to fix it :(

**Comment by bee-san** [comment]
I'm going to have to squash the commits via GitHub when it merges, it's very hard for me to squash them for some reason. Following this:
https://stackoverflow.com/questions/5667884/how-to-squash-commits-in-git-after-they-have-been-pushed
(not rebase but reset, the 2nd answer) causes it to reset to a non-working version which I have to fix again :(

File: src/scanner/main.rs [hunk 3]
```diff
@@ -0,0 +1,476 @@
+extern crate shell_words;
```
Comment (bee-san) [inline src/scanner/main.rs:1]: It's annoying that this entire file is classes as "new" but I don't know how to fix it :cry:

Comment (bernardoamc) [reply src/scanner/main.rs:1]: I can fix it for you, no worries! :)

Comment (bernardoamc) [reply src/scanner/main.rs:1]: Your local master is probably out of sync with the remote master.

**Pull request closed without merging**